common/lib/common-utils/package.json

```diff
@@ -76,6 +76,7 @@
 	},
 	"dependencies": {
 		"@fluidframework/common-definitions": "^0.20.1",
+		"@fluidframework/core-interfaces": "workspace:~",
```

Comment:
this is invalid. this must remain on common-def. you can't reference a client package from common